manifest/crds/sparkoperator.k8s.io_scheduledsparkapplications.yaml

```diff
@@ -3668,6 +3668,10 @@ spec:
                   type: string
                 sparkUIOptions:
                   properties:
+                    serviceAnnotations:
```

Comment:
The CRD definitions in the chart folder also needs to get updated.

Reply:
Done. Is there any version update required?